gemfiles/rails5.gemfile

```diff
@@ -0,0 +1,53 @@
+source 'http://rubygems.org'
+
+gem 'appraisal', '2.1.0'
+gem 'rails', '~> 5.0.0'
+gem 'roqua-support', git: 'git@github.com:roqua/roqua-support.git', branch: 'dd-rails-5'
+
+gem 'sprockets-rails', '2.3.3'
+
+# Blueprint is no longer a part of compass
+gem 'compass-blueprint'
+gem 'compass-rails', '3.0.2'
+
+gem 'susy'
+
+gem 'jquery-rails'
```

Comment:
Deze file hoort gegenereerd te worden door appraissal. Dingen die in de gemspec staan komen eigenlijk nooit terug in de gemfile. 
Specifieke dependencies zet je in de Appraisals file.